### DISCUSSION TITLE
onControlsVisibilityChange

### DISCUSSION THREAD

**Discussion by YoungYanchi** [TheWidlarzGroup/react-native-video#3936] [Ideas]
### What problem are you trying to solve?

On Android there is an `onControlsVisibilityChange` function, but there is no equivalent way to get the controls hiding event on iOS. The user is looking for a workaround to detect when the controls are hidden on iOS.

### Proposed solution

Provide a way (e.g. an `onControlsVisibilityChange` event/callback) to get the controls visibility / hiding event on iOS, matching the behavior already available on Android.

### Platforms

iOS, Android

**Comment by KrzysztofMoch** [comment]
Unfortunately there is no event for that on iOS so we can't do anything about it 😕
